include/spdlog/details/spdlog_impl.h

```diff
@@ -31,6 +31,7 @@
 #include "../sinks/file_sinks.h"
 #include "../sinks/stdout_sinks.h"
 #include "../sinks/syslog_sink.h"
+#include "../sinks/android_sink.h"
```

Comment:
@gabime Okay, so to clarify: I'm removing this line and it means that if user want to use android sink he should include header manually, like `#include <spdlog/sinks/android_sink.h`. Guess it would be nice to document it somewhere (?)